cumulus/pallets/dmp-queue/src/lib.rs

```diff
@@ -146,9 +146,6 @@ pub mod pallet {
 		Completed { error: bool },
 	}
 
-	#[pallet::call]
-	impl<T: Config> Pallet<T> {}
```

Comment:
any impact?

Reply:
At some point i thought it reduced the number of variants that we have to match on in the `BaseCallFilter`, now am not sure anymore but it also does not introduce issues.